dashboard/charts.py

```diff
@@ -11,7 +11,8 @@ def get_most_copies_sold_chart(data: DataFrame) -> alt.Chart:
     """Returns a bar chart of most copies sold when passed tracks/albums data."""
 
     return alt.Chart(data).mark_bar().encode(
-        x=alt.X("title", title="Title", axis=alt.Axis(labelAngle=-45)),
+        x=alt.X("title", title="Title", axis=alt.Axis(
+            labelAngle=-45)).sort("-y"),
```

Comment:
Why was `.sort("-y")` added?

Reply:
To make the bar graphs look nicer by sorting the bars in order of most sales to least sales

Reply:
Nice!